src/nvim/version.c

```diff
@@ -2218,7 +2218,7 @@ void intro_message(int colon)
   int sponsor;
   char *p;
   static char *(lines[]) = {
-    N_(NVIM_VERSION_LONG),
+    N_(NVIM_VERSION_LONG),  // NOLINT(bugprone-suspicious-missing-comma)
```

Comment:
shouldn't this be done at the definition, i.e. here: https://github.com/neovim/neovim/blob/ba887da97708545f1174e387b4807119ad5a40f6/src/nvim/version.c#L37

Reply:
Doesn't matter, both works. I don't think using `//` is necessary, but it should work as well.